grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -33,12 +33,18 @@
 
     <import resource="client-batch-job-context.xml"/>
 
+    <import resource="client-validateJob.xml" />
     <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
     <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
 
+        <batch:decision id="validateJob" decider="validJobDecider">
+            <next on="VALID" to="clientWorkflowOff" />
+            <fail on="INVALID" exit-code="VALIDATION_FAILED" />
+        </batch:decision>
+
```

Comment:
@jbornemann I think -1 testing.
This decider is not getting executed when I run the job. Tested with a sample yaml file (from local pub to author) like this -- 

```yaml
serverHost: publish.twcable.local
serverPassword: admin
serverPort: 4503
serverUsername: admin
deltaContent: true
pathConfigurations: 
  - 
    path: /content/a
  - 
    path: /content/b

```

I expected the Client (local author) to fail without even doing an http call to the Server (local publish) but that was not the case.